test_scripts/Capabilities/PersistingHMICapabilities/015_SDL_does_not_override_invalid_language_in_HMICapabilityFile_file.lua

```diff
@@ -0,0 +1,33 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0249-Persisting-HMI-Capabilities-specific-to-headunit.md
+--
+-- Description: Check that SDL does not override in "hmi_capabilities_cache.json" file
+-- in case HMI sends TTS/VR/UI.OnLanguageChange notification with invalid language
+--
+-- Preconditions:
+-- 1  Value of HMICapabilitiesCacheFile parameter is defined (hmi_capabilities_cache.json) in smartDeviceLink.ini file
+-- 2. HMI capabilities cache file (hmi_capabilities_cache.json) doesn't exist on file system
+-- 3. SDL and HMI are started
+-- 4. HMI sends all HMI capabilities to SDL
+--
+-- Sequence:
+-- 1. HMI sends "TTS/VR/UI.OnLanguageChange" notifications with invalid language to SDL
+--   a. SDL does not override TTS/VR/UI.language in HMI capabilities cache file ("hmi_capabilities_cache.json")
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Capabilities/PersistingHMICapabilities/common')
+
+--[[ Local Variable ]]
+local invalidLanguage = "EN-EN"
```

Comment:
change the name of invalidLanguage to unsupportedlanguage

Reply:
Done ( see [20c8f86](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2416/commits/20c8f86f9e8195f5a5858c31fa63ff0f7fab82c3) )